R/DataDiscoveryRetrieval.R

```diff
@@ -52,14 +56,32 @@
 #' tada1 <- TADAdataRetrieval(statecode = "WI",
 #'                            countycode = "Dane",
 #'                            characteristicName = "Phosphorus")
+#' 
+#' tada2 <- TADAdataRetrieval(ProjectIdentifier = "Anchorage Bacteria 20-21")
+#' 
+#' tada3 <- TADAdataRetrieval(statecode = "UT", 
+#'                            characteristicName = c("Ammonia", "Nitrate", "Nitrogen"), 
+#'                            startDate = "10-01-2020")
+#' 
+#' tada4 <- TADAdataRetrieval(statecode = "SC", countycode  = "Abbeville")
+#' 
+#' # countycode queries require a statecode
+#' tada5 <- TADAdataRetrieval(countycode = "US:02:020")
+#' 
+#' tada6 <- TADABigdataRetrieval(startDate = "2019-01-01", 
```

Comment:
Should change date format so it is consistent across examples. Let me know if you'd like me to do this on my next branch commit.